src/components/views/read/cvForm/templates/FirstTemplate.tsx

```diff
@@ -0,0 +1,310 @@
+              }}
+            >
+              {personalData.gender} - {personalData.birthDate}
+            </Text>
+            <Text
+              style={{
+                paddingTop: '10px',
+                paddingBottom: '5px',
+                fontSize: '11px',
+                color: 'white',
+              }}
+            >
+              {personalData.address}
+            </Text>
+          </View>
+          <View
+            style={{
+              marginTop: '10px',
+              width: '50%',
+              height: '1px',
+              backgroundColor: '#FFF',
+              textAlign: 'center',
+            }}
+          />
+          <Text
+            style={{
+              paddingTop: '10px',
+              paddingBottom: '5px',
+              fontSize: '10px',
+              color: 'white',
+            }}
+          >
+            {personalData.emailAddress}
+          </Text>
+          <Text
+            style={{
+              paddingTop: '10px',
+              paddingBottom: '5px',
+              fontSize: '14px',
+              color: 'white',
+            }}
+          >
+            +{personalData.phoneNum}
+          </Text>
+          <Text
+            style={{
+              paddingTop: '10px',
+              paddingBottom: '5px',
+              fontSize: '14px',
+              color: 'white',
+            }}
+          >
+            {personalData.profileLinks}
+          </Text>
+          <Text
+            style={{
+              paddingTop: '10px',
+              paddingBottom: '5px',
+              fontSize: '14px',
+              color: 'white',
+            }}
+          >
+            {personalData.personalLink}
+          </Text>
+        </View>
+      </View>
+      {/* RIGHT SECTION OF THE CV */}
+      <View
+        style={{
+          margin: 10,
+          padding: 10,
+          paddingTop: 20,
+          width: '75%',
+        }}
+      >
+        <Text>Education</Text>
+        <View
+          style={{
+            height: '1px',
+            marginTop: '13px',
+            marginBottom: '12px',
+            width: '100%',
+            backgroundColor: '#084c41',
+          }}
+        />
+        {educationData.map((item, index) => (
+          <View key={index} style={{ marginBottom: 10 }}>
+            <Text style={{ fontSize: '13px' }}>
+              {item.major} - {item.institutionName}
+            </Text>
+            <Text
+              style={{ fontSize: '9', color: '#959ba6', paddingBottom: '5' }}
+            >
+              {item.startDate} - {item.endDate}
+            </Text>
+            {/* <Text style={{ fontSize: '11px' }}>Degree: {item.degree}</Text> */}
+            {Object.entries(item)
+              .filter(
+                ([key]) =>
+                  key !== 'id' &&
+                  key !== 'major' &&
+                  key !== 'institutionName' &&
+                  key !== 'startDate' &&
+                  key !== 'endDate',
```

Comment:
Use the `Array.includes` function for this instead of enumerating the keys you do not want.

Reply:
fixed